lib/api/v3/work_packages/work_package_representer.rb

```diff
@@ -338,13 +352,6 @@ def version
           Versions::VersionRepresenter.new(represented.fixed_version, current_user: current_user)
         end
 
-        def custom_properties
-          values = represented.custom_field_values
-          values.map do |v|
-            { name: v.custom_field.name, format: v.custom_field.field_format, value: v.value }
-          end
-        end
```

Comment:
This will probably break the front end. @NobodysNightmare, shall we tell @0xF013 or shall we surprise him?